provider/pkg/gen/python-templates/apiextensions/CustomResource.py

```diff
@@ -11,8 +12,15 @@
 
 
 class CustomResource(pulumi.CustomResource):
-    def __init__(self, resource_name, api_version, kind, spec=None, metadata=None, opts=None,
-                 __name__=None, __opts__=None):
+    def __init__(self,
+                 resource_name: str,
+                 api_version: str,
+                 kind: str,
+                 spec: Optional[pulumi.Input[Any]] = None,
+                 metadata: Optional[pulumi.Input[Any]] = None,
+                 opts: Optional[pulumi.ResourceOptions] = None,
+                 __name__=None,
```

Comment:
Just here to complain about the discrepancy in python's whitespacing convention when there's a type annotation + default value vs. just a default value. Thanks 🐍